ui/ducks/send/send.js

```diff
@@ -1937,6 +1951,38 @@ export function updateRecipient({ address, nickname }) {
   };
 }
 
+/**
+ * This method is called when user changes the network in the send page.
+ * It doesn't update the chain id for transaction, rather it reject the existing and create new transaction instead
+ * then it assigns the new transaction id to the draft transaction id.
+ */
+export function onNetworkChange() {
```

Comment:
It's my mistake to forgot mentioning in the PR description.

Since this PR is going to be merged to `develop`, this is not used anywhere in the code because network switcher is already disabled.

I've created another branch to manually test this and this function is being used there https://github.com/MetaMask/metamask-extension/commit/fda9a6c7947a8ca7bd8e412c41aa03a56c7862ac

If we merge this PR by getting approval from extension team, then we will merge it and they will use it, that was my plan.

Reply:
Pushed changes into multichain components now